conanfile.py

```diff
@@ -28,19 +29,22 @@ class CuraEngineConan(ConanFile):
         "enable_benchmarks": [True, False],
         "enable_extensive_warnings": [True, False],
         "enable_plugins": [True, False],
+        "enable_sentry": [True, False],
         "enable_remote_plugins": [True, False],
     }
     default_options = {
         "enable_arcus": True,
         "enable_benchmarks": False,
         "enable_extensive_warnings": False,
         "enable_plugins": True,
+        "enable_sentry": False,
```

Comment:
Should we also update the gh runner install commands to `enable_sentry` is this already done, or should we still do this?

Reply:
That's a bit of code that @jellespijker added. I think that it should work, but I haven't tried it.

Reply:
https://github.com/Ultimaker/Cura/commit/2f452c6581ba11c3381b44ff1d5151fc56173687